.chloggen/vcenterreceiver_fix_multiple_datacenters.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: vcenterreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: vcenterreceiver fixes errors when multiple datacenters in some of the client calls.
```

```suggestion
note: Fixes errors when multiple datacenters in some of the client calls.
```
The component name from `component` is included with the changelog note in releases, so we don't need to include it here as well.